### DISCUSSION TITLE
add the fly over box info with mouse

### DISCUSSION THREAD

**Discussion by joc2** [egzumer/uvk5-chirp-driver#38] [Show and tell]
In chirp driver add the fly over with the mouse to describe function, ( a help directly in application )
In the integrate version driver chirp too.

take a look  and think about it.

i can do it, but i dont have access any more..

![image](https://github.com/egzumer/uvk5-chirp-driver/assets/56229329/74d4a682-79d6-47ae-8069-503bf5e5e413)

**Comment by egzumer** [comment]
Everyone has access, including you. Make a PR (push request), or push to seperate branch. Also the actual driver is in the chirp repository, I have the same access rights there as you do.
